tests/unit/frameworks/test_any_agent.py

```diff
@@ -39,49 +38,6 @@ def create_agent_with_model_args(framework: AgentFramework) -> AnyAgent:
     )
 
 
-async def mock_streaming_response(
-    *args: Any, **kwargs: Any
-) -> AsyncGenerator[dict[str, Any], None]:
-    """
-    Mock the streaming response from litellm's acompletion function.
-    Accepts all arguments that would be passed to the real acompletion function.
-    """
-    # First chunk with role
-    yield {
-        "choices": [
-            {
-                "delta": {"role": "assistant", "content": "The state "},
-                "index": 0,
-                "finish_reason": None,
-            }
-        ]
-    }
-
-    # Middle chunks with content
-    yield {
-        "choices": [
-            {"delta": {"content": "capital of "}, "index": 0, "finish_reason": None}
-        ]
-    }
-
-    yield {
-        "choices": [
-            {
-                "delta": {"content": "Pennsylvania is "},
-                "index": 0,
-                "finish_reason": None,
-            }
-        ]
-    }
-
-    # Final chunk with finish reason
-    yield {
-        "choices": [
-            {"delta": {"content": "Harrisburg."}, "index": 0, "finish_reason": "stop"}
-        ]
-    }
-
-
 class TestAgentCreation:
```

Comment:
💅 let's drop the class in favor of functions for consistency

Reply:
If we want to separate creation from model args, let's put them in separate `test_` files